lib/node/asar-fs-wrapper.ts

```diff
@@ -308,21 +310,21 @@ export const wrapFsWithAsar = (fs: Record<string, any>) => {
 
     const archive = getOrCreateArchive(asarPath);
     if (!archive) {
-      if (shouldThrowStatError(options)) {
+      if (options?.throwIfNoEntry === true) {
         throw createError(AsarError.INVALID_ARCHIVE, { asarPath });
       };
       return null;
     }
 
     const stats = archive.stat(filePath);
     if (!stats) {
-      if (shouldThrowStatError(options)) {
+      if (options?.throwIfNoEntry === true) {
         throw createError(AsarError.NOT_FOUND, { asarPath, filePath });
       };
       return null;
     }
 
-    return asarStatsToFsStats(stats);
+    return asarStatsToFsStats(stats, options);
```

Comment:
Question is, why it not used native lstat for compute values, like:

```suggestion
    return lstat(pathArgument, options, callback);
```